Bottom: f734d98e6105ee950f7a199e72b06e09e2579d64
Top:    3364fe6dd47894ba374351c71bd944e8d8b76eae
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-08-18 16:11:01 +0800

Refresh of tun-rename-generic_xdp-to

---

diff --git a/drivers/net/tun.c b/drivers/net/tun.c
index 0fe5c91..54be759 100644
--- a/drivers/net/tun.c
+++ b/drivers/net/tun.c
@@ -1357,7 +1357,7 @@ static struct sk_buff *tun_build_skb(struct tun_struct *tun,
 
 	if (xdp_xmit) {
 		skb->dev = tun->dev;
-		skb_xdp_tx(skb, xdp_prog);
+		generic_xdp_tx(skb, xdp_prog);
 		rcu_read_lock();
 		return NULL;
 	}
